qiskit/providers/models/backendconfiguration.py

```diff
@@ -290,6 +360,20 @@ def __init__(self,
                  meas_kernels: List[str],
                  discriminators: List[str],
                  hamiltonian: Dict[str, str] = None,
+                 channel_bandwidth=None,
```

Comment:
What about `PulseHamiltonianSchema`? Is it no longer defined because hamiltonian is already passed in as a dictionary anyway?

Reply:
Yeah, since there was no model object for it the schema object was only being used for checking the dict. I can mirror the structural validation it was doing, but it seemed like unnecessary overhead since it's returned from the device and not something we'll actually ever need to validate